whois-api/src/test/java/net/ripe/db/whois/api/mail/dequeue/MessageServiceTestIntegration.java

```diff
@@ -90,6 +90,22 @@ public void testBouncedEmailFromCorrectEmail() throws MessagingException, IOExce
         assertThat(isUndeliverableAddress(BOUNCED_MAIL_RECIPIENT), is(true));
     }
 
+    @Test
+    public void testBouncedMultipleEmailFromCorrectEmail() throws MessagingException, IOException {
+        insertOutgoingMessageId("XXXXXXXX-5AE3-4C58-8E3F-860327BA955D@ripe.net", BOUNCED_MAIL_RECIPIENT);
+        insertOutgoingMessageId("XXXXXXXX-5AE3-4C58-8E3F-860327BA955D@ripe.net", ANOTHER_BOUNCED_MAIL_RECIPIENT);
```

Comment:
yes, we check that the recipients in the email that we receive exists in the database for that specific message_id.
If not anyone can send us any message with the recipients that they want and we can be setting those recipients as unsubscribed by mistake (because that "anyone" sent to us a fake mail)
So we are checking that the message_id exist and that the recipients of that message exist in our database for that specific message Id

Reply:
so I need to add this information in the outgoing table for bypass this check